src/Admin/Extension/CreateSnapshotAdminExtension.php

```diff
@@ -40,6 +40,10 @@ public function postPersist(AdminInterface $admin, object $object): void
 
     public function postRemove(AdminInterface $admin, object $object): void
     {
+        if ($object instanceof PageInterface) {
+            return;
+        }
+
         $this->createByPage($object);
```

Comment:
I'm not familiar with this lib, maybe you can explain to me the change behind ?

SOmething like:
"This extension is used to create a snapshot of a Page. When you persist/update a Page or a PageBlock you want to update the snapshot of the Page. But when you remove something, the snapshot should only be updated if we remove a PageBlock. When we removed a Page, the snapshot should just be deleted with the page" ?

Reply:
It works on my side. It prevent the snapshot to re-create the page on page deletion.